<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="table_showAllScientists">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sex</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Birth Year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Died in </string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_add_scientist">
      <property name="text">
       <string>Add a new Scientist</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_input_label">
        <property name="text">
         <string>Input scientist name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_error_name">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="Input_Scientist_Name"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_input_sex">
        <property name="text">
         <string>Sex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_error_sex">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="input_sex"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_year_of_birth">
        <property name="text">
         <string>Year of birth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_error_YoB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="input_year_of_birth"/>
    </item>
    <item>
     <widget class="QLabel" name="label_year_of_death">
      <property name="text">
       <string>Year of death</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_year_of_death"/>
    </item>
    <item>
     <widget class="QPushButton" name="button_scientists_to_table">
      <property name="text">
       <string>Add to table</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
